vllm/distributed/kv_transfer/kv_connector/v1/decode_bench_connector.py

```diff
@@ -0,0 +1,305 @@
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: Copyright contributors to the vLLM project
+"""
+DecodeBenchConnector: A KV Connector for decode instance performance testing.
+
+This connector emulates a prefill-decode disaggregated setting by filling
+the KV cache with dummy values, allowing measurement of decoder performance
+under larger input sequence lengths (ISL) in resource-limited environments.
```

Comment:
I wonder if this kind of dev stuff shouldn't be kept under tests/ or benchmarks/ and registered as an OOT connector